app/src/main/java/org/p2p/wallet/home/MainFragment.kt

```diff
@@ -79,7 +79,10 @@ class MainFragment : BaseFragment(R.layout.fragment_main), MainTabsSwitcher, Cen
         }
         deeplinksManager.handleSavedDeeplinkIntent()
 
-        onCreateActions.forEach(::doOnCreateAction)
+        if (onCreateActions.isNotEmpty()) {
```

Comment:
no need for that, forEach won't invoke if array is empty

Reply:
i know but i do not want to call all this block if we are empty like onCreateActions = emptyArray()